web_external/templates/body/uploadDataset.jade

```diff
@@ -0,0 +1,101 @@
+
+  .isic-form-container
+    .isic-form-group
+      h3 Basic information
+      p.
+        Enter the name of the dataset and add a detailed description.
+      .form-group
+        label(for="isic-dataset-name") Name
+        input#isic-dataset-name.form-control.input-sm(
+          type="text")
+      .form-group
+        label.control-label(for="isic-dataset-description") Description
+        textarea.input-sm#isic-dataset-description.form-control
+    .isic-form-group
+      h3 Upload files
+      ul.nav.nav-tabs
+        li#isic-upload-images.active
+          a Upload images (.zip)
+        li#isic-upload-metadata
+          a Upload metadata (.csv)
+      .isic-upload-container
+        .isic-upload-widget-container
+        .isic-upload-reset-container
+           .isic-upload-list
+           button#isic-upload-reset.btn.btn-secondary.btn-md Reset
+    .isic-form-group
+      h3 Licensing
+      p.
+        Select the license for the dataset and enter your name to
+        electronically sign the contributor license agreement.
+      .form-group
+        label.control-label(for="isic-dataset-license") License
+        select#isic-dataset-license.form-control.input-sm
+          option(value="CC-0")
+            | CC-0 (No restrictions)
+          option(value="CC-BY")
+            | CC-BY (Attribution)
+          option(value="CC-BY-NC")
+            | CC-BY-NC (Attribution-NonCommercial)
+        .isic-dataset-agreement-link-container
+          a.isic-dataset-agreement-link
+            | Help me choose
+      .form-group
+        label.control-label(for="isic-dataset-agreement")
+          | Contributor license agreement
+        textarea.input-sm#isic-dataset-agreement.form-control(readonly).
+
+          Lorem ipsum dolor sit amet, eos te aliquid gubergren, veniam perpetua voluptatum
+          ad usu, sed an dico choro inermis. Ei nam elit audiam, in luptatum explicari
+          pri. Tale nostro abhorreant ex qui, graeci tibique mel no. Vix vide malis mazim
+          an, vix postea recteque qualisque ei.
+
+          Id scripta concludaturque vix, ut sonet verear legendos per. Falli graece
+          noluisse est cu. Diam epicurei recusabo usu ut. Justo munere detracto sed ne,
+          qui nemore voluptatibus cu. Ius te homero honestatis.
+
+          Vitae oportere euripidis pro ad, has an epicuri prodesset repudiandae. Ea usu
+          nihil instructior. Omnes prompta vix id, prodesset pertinacia an pri. Vocibus
+          fastidii vituperatoribus cu vim, liber neglegentur mea ei. Ne sit epicuri
+          noluisse tractatos, per ex quaeque salutatus.
+
+          Per ad odio definiebas, id justo dolores appareat vim. No altera tacimates pro,
+          et sea brute altera detracto. Eam ad dicant quodsi instructior. Et eligendi
+          accusata imperdiet mel, duo at esse voluptaria. Quo quot graeco no.
+
+        .isic-dataset-agreement-link-container
+          a.isic-dataset-agreement-link
```

Comment:
I guess we'll add behavior for these buttons once we get the final draft. That's fine.